packages/scripts/utils/config.js

```diff
@@ -202,7 +203,7 @@ function getWebpackEntryPoints() {
 						const filepath = join(
 							dirname( blockMetadataFile ),
 							value.replace( 'file:', '' )
-						);
+						).replace( /\\/g, '/' );
```

Comment:
Does it mean that it works incorrectly on Windows at `split` call with `src/`? Mayb we could use [`path.sep`](https://nodejs.org/api/path.html#pathsep) instead of `/`?

Reply:
`Does it mean that it works incorrectly on Windows at split call with src/?`

Yes.
I couldn't find any official documentation, but it seems to be converted to a backslash on Windows when passing through `path.join`.

Like this:

```javascript
const filepath = join(
	dirname( blockMetadataFile ),
	value.replace( 'file:', '' )
);
console.log( dirname( blockMetadataFile ) );
console.log( filepath );
```

Output: 
```javascript
D:/path/to/gutenberg/gutennpride/src
D:\path\to\gutenberg\gutennpride\src\index.js
```

Therefore, we need to replace the separator in the `filepath` variable with a forward slash.

Alternatively, the following will also work.

```javascript
const [ , entryName ] = filepath
	.split( '.' )[ 0 ]
	.split( `src${ sep }` );
```

Reply:
For now, let’s use what you already have working 😃